src/App.tsx

```diff
@@ -2,6 +2,7 @@ import { PeoplePage } from './components/PeoplePage';
 import { Navbar } from './components/Navbar';
```

Comment:
Would be better to create a separate 'Layout' or 'Base' component for such common ...well, layout. 
If you want to share some layout between all your page components, this should be done like this: 

```
export const Layout = () => {
  return (
    <div data-cy="app">
      <Navbar />

      <main className="section">
        <div className="container">
          <Outlet />
        </div>
      </main>
    </div>
  );
};
```

And used in routes like: 

```
      <Route path="/" element={<Layout />}>
        <Route path="people">
          <Route path=":personSlug?" element={<PeoplePage />} />
        </Route>
      </Route>
```